app/Http/Controllers/Admin/PageController.php

```diff
@@ -0,0 +1,47 @@
+<?php namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Http\Services\ContractService;
+use App\Http\Services\SubscriberService;
+
+/**
+ * Class PageController
+ * @property SubscriberService subscriber
+ * @property ContractService   contract
+ * @package App\Http\Controllers\Admin
+ */
+class PageController extends Controller
+{
+    /**
+     * PageController constructor.
+     *
+     * @param SubscriberService $subscriber
+     * @param ContractService   $contract
+     */
+    public function __construct(SubscriberService $subscriber, ContractService $contract)
+    {
+        $this->middleware('user');
+        $this->subscriber = $subscriber;
+        $this->contract   = $contract;
+    }
+
+    /**
+     * Returns subscribers
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     */
+    public function index()
+    {
+        $subscribers = $this->subscriber->getSubscribers();
```

Comment:
rename getSubscribers to paginate method and also implement pagination and filter by subscribe/unsubscribe.